macros/elvish_macros.cfg

```diff
@@ -159,22 +191,70 @@
         #po: The talk between Kyobaine as Elvish Druid and Gawen, during winter
         {MESSAGE (Kyobaine) (portraits/kyobaine.png) (Kyobaine) _"Oooh, no. I refuse to fight in these conditions, as long as I'm clad as such."}
         {MSGW_Gawen _"What do you mean? Your clothes are just fine."}
-        # TODO: what if she put on snowshoes in a previous scenario?
-        {MESSAGE (Kyobaine) (portraits/kyobaine.png) (Kyobaine) _"<i>For the spring</i>. The least you can do is to give me proper shoes. You expect me to walk in this snow <i>barefoot</i>??"}
-        {MSGW_Gawen _"That's ridiculous."}
-        {MESSAGE (Kyobaine) (portraits/kyobaine.png) (Kyobaine) _"I couldn't agree more. Call me when you have new shoes."}
+        [if]
+            [have_unit]
+                id=Kyobaine
+                trait=snow_shoes
+            [/have_unit]
+            [then]
+                #po: Kyobaine asks Gawen for different clothes when she's wearing snowshoes:
+                {MESSAGE (Kyobaine) (portraits/kyobaine.png) (Kyobaine) _"Except for the snowshoes, these are clothes <i>for the spring</i>. The least you can do is to give me a proper cloak. You expect me to fight when I'm shivering and have goosebumps??"}
+                {MSGW_Gawen _"That's ridiculous."}
+                #po: Kyobaine asks Gawen for different clothes when she's wearing snowshoes:
+                {MESSAGE (Kyobaine) (portraits/kyobaine.png) (Kyobaine) _"I couldn't agree more. Call me when you have a new cloak."}
+            [/then]
+            [else]
+                #po: the old version of Kyobaine's request to Gawen:
+                {MESSAGE (Kyobaine) (portraits/kyobaine.png) (Kyobaine) _"<i>For the spring</i>. The least you can do is to give me proper shoes. You expect me to walk in this snow <i>barefoot</i>??"}
+                {MSGW_Gawen _"That's ridiculous."}
+                #po: the old version of Kyobaine's request to Gawen:
+                {MESSAGE (Kyobaine) (portraits/kyobaine.png) (Kyobaine) _"I couldn't agree more. Call me when you have new shoes."}
+            [/else]
+        [/if]
 
         {MSGOPTI2 (Gawen Hagarthen) (portraits/gawen_winter.png) (_"Well...")
         (_"All right, how about I give you 25 gold pieces instead of 20?") (yes)
         (_"All right, we will wait for the spring.") (no)}
         {IF ano_opt equals yes}
         {MESSAGE (Kyobaine) (portraits/kyobaine.png) (Kyobaine) _"You think you can bribe me, an elvish druid, with gold? Oh well. This will allow me to buy better winter clothes."}
-        # TODO: come up with new art to show Kyobaine's better winter clothes
 
```

Comment:
So, I kind of wanted to go even further with the banter here, but decided to hold myself back... @nemaara WDYT about me possibly including additional banter here in a future PR?

Reply:
Go for it. Just don't add delays so people can hit esc if they want.